rpc/amqp-rpc.go

```diff
@@ -154,8 +221,16 @@ func (rpc *AmqpRPCServer) Start() (err error) {
 				rpc.log.Audit(fmt.Sprintf(" [s<][%s][%s] Misrouted message: %s - %s - %s", rpc.serverQueue, msg.ReplyTo, msg.Type, core.B64enc(msg.Body), msg.CorrelationId))
 				continue
 			}
-			response := cb(msg.Body)
-			rpc.log.Info(fmt.Sprintf(" [s>][%s][%s] replying %s(%s) [%s]", rpc.serverQueue, msg.ReplyTo, msg.Type, core.B64enc(response), msg.CorrelationId))
+			var response RPCResponse
+			response.ReturnVal, err = cb(msg.Body)
+			response.Error = wrapError(err)
+			jsonResponse, err := json.Marshal(response)
+			if err != nil {
+				// AUDIT[ Error Conditions ] 9cc4d537-8534-4970-8665-4b382abe82f3
+				rpc.log.Audit(fmt.Sprintf(" [s>][%s][%s] Error condition marshalling RPC response %s(%s) [%s]", rpc.serverQueue, msg.ReplyTo, msg.Type, "", msg.CorrelationId))
```

Comment:
I feel like we shouldn't print the response body (in parens) as "" here unconditionally, either we should remove that arg (probably fine), or try to print `response.ReturnVal`